src/platforms/javascript/guides/capacitor/dsym.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Uploading Debug Symbols (iOS)
+sidebar_order: 20
```

Comment:
I'm not sure where this should fall in the TOC order. Maybe @bruno-garcia can advise on that.